### PR TITLE
fix: refine migrate gvg task workflow

### PR DESCRIPTION
**Pull request by will-2012**
### Description

Refine migrate gvg task workflow.

### Rationale

1. Infinitely retry migration tasks in case of failure, only be gc when it is finished.
2. When the task is not updated within a certain period of time, it will be executed by other executors.

### Example

N/A.

### Changes

Notable changes: 
* Refine migrate gvg task retry policy and gc policy.
* Refine some log and add metrics.

### CODE REVIEW

**Review by sysvm — APPROVED** [review]
LGTM

File: base/gfspapp/task_options.go [hunk 1]
```diff
@@ -52,9 +52,9 @@ const (
 	// MaxMigratePieceTime defines the max timeout to migrate piece.
 	MaxMigratePieceTime int64 = 50
 	// MinMigrateGVGTime defines the min timeout to migrate gvg.
-	MinMigrateGVGTime int64 = 10
+	MinMigrateGVGTime int64 = 1800 // 0.5 hour
 	// MaxMigrateGVGTime defines the max timeout to migrate gvg.
-	MaxMigrateGVGTime int64 = 50
+	MaxMigrateGVGTime int64 = 3600 // 1 hour
```
Comment (Fynnss) [inline base/gfspapp/task_options.go:57]: Is the 1 hour timeout also for GVGs with large data storage?

Comment (will-2012) [reply base/gfspapp/task_options.go:57]: if the task is running, the update time will be updated, and will not be judged as timed out ( update_time + this var > now).

Comment (Fynnss) [reply base/gfspapp/task_options.go:57]: In other words, will the start time of the timeout be updated?

Comment (Fynnss) [reply base/gfspapp/task_options.go:57]: > if the task is running, the update time will be updated, and will not be judged as timed out ( update_time + this var > now).

Okay, make sense.

**Review by Fynnss — APPROVED** [review]
LGTM

**Pull request merged**